### PR TITLE
RNS SDK for Subdomains

### PR DESCRIPTION
**Pull request by OnyekaIjeh**
This PR allows the RNS React manager to use the RNS SDK for registering subdomains.

### CODE REVIEW

**Review by LordCheta — APPROVED** [review]
LGTM

File: src/app/rns-sdk.js [hunk 1]
```diff
@@ -50,4 +49,4 @@ export const rns = (
 
 export const resolver = (
   signer = defaultSigner,
-) => new AddrResolver(resolverAddress, signer);
+) => new AddrResolver(registryAddress, signer);
```
Comment (sergio-jara) [inline src/app/rns-sdk.js:52]: just add a comment behind the reasoning behind this lol

File: src/app/tabs/newAdmin/subdomains/operations.js [hunk 2]
```diff
@@ -43,33 +46,38 @@ const registerSubdomain = (
...
   dispatch(waitingNewSubdomainConfirm());
-  const transactionConfirmed = result => () => {
+
+  const signer = await getSigner();
+
+  const r = rns(signer);
+
+  try {
+    // eslint-disable-next-line no-debugger
+    debugger;
```
Comment (sergio-jara) [inline src/app/tabs/newAdmin/subdomains/operations.js:56]: remove

File: src/app/tabs/newAdmin/subdomains/operations.js [hunk 3]
```diff
@@ -43,33 +46,38 @@ const registerSubdomain = (
...
 
 const getSubdomainOwner = (domain, subdomain) => async (dispatch) => {
   const hash = namehash(`${subdomain}.${domain}`);
 
-  const rns = getRNS();
-  await rns.compose();
-  await rns.contracts.registry.methods.owner(hash).call((error, result) => {
+  const r = getRNS();
+  await r.compose();
+  await r.contracts.registry.methods.owner(hash).call((error, result) => {
```
Comment (sergio-jara) [inline src/app/tabs/newAdmin/subdomains/operations.js:80]: this was not possible using the SDK?

File: src/app/tabs/newAdmin/subdomains/operations.js [hunk 4]
```diff
@@ -165,9 +177,9 @@ export const setSubdomainOwner = (
   const label = `0x${sha3(normalize(subdomain))}`;
   const node = namehash(parentDomain);
 
-  const rns = getRNS();
-  await rns.compose();
-  rns.contracts.registry.methods.setSubnodeOwner(node, label, newAddress)
+  const r = getRNS();
+  await r.compose();
+  r.contracts.registry.methods.setSubnodeOwner(node, label, newAddress)
```
Comment (sergio-jara) [inline src/app/tabs/newAdmin/subdomains/operations.js:182]: and this?

**Review by sergio-jara — CHANGES_REQUESTED** [review]

**Comment by sergio-jara** [comment]
We should be running the linter before pushing please

File: src/app/tabs/newAdmin/subdomains/operations.js (hunk 3 above)
Comment (OnyekaIjeh) [reply src/app/tabs/newAdmin/subdomains/operations.js:80]: These are not part of the flow, will make the change in another issue.

File: src/app/tabs/newAdmin/subdomains/operations.js (hunk 4 above)
Comment (OnyekaIjeh) [reply src/app/tabs/newAdmin/subdomains/operations.js:182]: These are not part of the flow, will make the change in another issue.

File: src/app/tabs/newAdmin/subdomains/operations.js (hunk 2 above)
Comment (OnyekaIjeh) [reply src/app/tabs/newAdmin/subdomains/operations.js:56]: Good catch.

File: src/app/rns-sdk.js (hunk 1 above)
Comment (OnyekaIjeh) [reply src/app/rns-sdk.js:52]: Okay, sure!

**Comment by OnyekaIjeh** [comment]
> We should be running the linter before pushing, please

I will open a PR to do this automatically after committing or pushing. 
This was in place before but maybe was removed due to some regression.

**Review by sergio-jara — APPROVED** [review]

**Pull request merged**